eland/ml/pytorch/transformers.py

```diff
@@ -77,7 +77,7 @@
     "text_similarity",
 }
 ARCHITECTURE_TO_TASK_TYPE = {
-    "MaskedLM": ["fill_mask", "text_embedding"],
+    "MaskedLM": ["fill_mask", "text_embedding", "text_expansion"],
```

Comment:
This map is used to infer the task type when the `--task-type` parameter is not passed on the command line. The `MaskedLM` architecture does not mean that the model can be used for text_expansion, for text_expansion the user will have to explicitly set `--task-type text_expansion` 

```suggestion
    "MaskedLM": ["fill_mask", "text_embedding"],
```